jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -341,13 +345,17 @@ static void afterClinitInitialization() {
 		} catch (IllegalArgumentException e) {
 			// use the defaultCharset()
 		}
-		/*[ELSE]*/
+		/*[ELSE] PLATFORM-mz31|PLATFORM-mz64 */
+		/*[IF JAVA_SPEC_VERSION >= 18]*/
+		consoleDefaultEncoding = props.getProperty("native.encoding"); //$NON-NLS-1$
```

Comment:
I feel this change and L334 are not required if https://github.com/eclipse-openj9/openj9/pull/14285 is merged.
Does the change fix any test failure?

Reply:
This doesn't have anything to do with https://github.com/eclipse-openj9/openj9/pull/14285

Reply:
I hope it fixes the earlier test failure https://openj9-jenkins.osuosl.org/view/Pull%20Requests/job/PullRequest-OpenJ9/1396/
The console is being created incorrectly on jdk18 as it's set to UTF-8 which won't match the actual console on the OS, unless it happens to be UTF-8 as well (which is usually the default for Linux, and is used on Mac).